src/sidebar/services/load-annotations.js

```diff
@@ -2,6 +2,14 @@
  * A service for fetching annotations, filtered by document URIs and group.
  */
 
+/**
+ * @typedef LoadAnnotationOptions
```

Comment:
Some further factoring would be ideal here and in `SearchClient` to structure options/parameters more cleanly.